### ISSUE TITLE
Handle exceptions from request API

### ISSUE THREAD

**Issue by mlooney** [emory-libraries/blacklight-catalog#699]
Exceptions from the hold request API should be handled, and the accompanying error messages returned to the user.

**Comment by mark-dce** [comment]
via @maxkadel 
```
Bad Request
401890 - 'User with identifier X of type Y was not found.'
401129 - 'No items can fulfill the submitted request.'
401136 - 'Failed to save the request: Patron has active request for selected item.'
60308 - 'Delivery to personal address is not supported.'
60309 - 'User does not have address for personal delivery.'
60310 - 'Delivery is not supported for this type of personal address.'
60325 - 'Fulfillment network institution is not valid.'
60326 - 'Fulfillment network pickup library or pickup circulation desk is not valid.'
60327 - 'Fulfillment network pickup library or pickup circulation desk is required.'
401652 - 'General Error - An error has occurred while processing the request.'
60330 - 'Invalid partial digitization volume or issue.'
60331 - 'Failed to create request.'
```

**Comment by eporter23** [comment]
This can be tested by requesting the same item twice.

**Comment by lovinscari** [comment]
I tried testing this morning by requesting MMSID 9937297878102486 and received an error message (on first request) "No items can fulfill the submitted request). I'm not sure if this was specific to that MMSID? I then tested 990001603580302486 and received the message, "Filed to save the request: Patron has active request for selected item." I'm assuming this is the expected behavior? 

I will submit a new ticket as I think the language needs to be a little more intuitive if possible? Also, I'd like to see at what point in RTA does a request show as active? For example, when I first requested the book and went in the second time to request again, logically I would have assumed the book as showing on hold but perhaps this is not completed until Access Services fulfills the request?
